api/controllers/user.controller.js

```diff
@@ -16,7 +16,7 @@ function getSingleUser (req, res) {
 
   User
     .findOne({ uid: uid })
-    .populate('orders.items.product')
+    .populate('orders.items.product basket.items.product')
```

Comment:
This bit is making the actual product information show instead of just `ObjectId('3098sf8908s09f8')`

https://github.com/jcolairo/masters2.0/blob/f4af0e42121b4aeb4151a123d0e6d4383d57c7e9/api/models/user.model.js

^ that is where the order gets moved from `basket` to `orders`